docs/api/5-changelog.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to the API will be documented here.
 
+## 13-12-2022
+
+No breaking changes, only additions and deprecations. The deprecated fields will be removed on 15.06.2023 at the earliest.
+
+### Lane numbers and traffic directions according to road link direction
+
+Added field `Lane.laneNumberAccordingToRoadLink`. It is now possible to retrieve lanes numbered according to the road link direction. (Traffic in odd-numbered lanes travels with the road link direction, while traffic in even-numbered lanes travels against the road link direction.) This matches the lane numbers used by NVDB, and they do not change over time. `Lane.laneNumberAccordingToMetering` is still available as before, with lane numbers according to the road's current metering direction. `Lane.laneNumber`is now deprecated, as its meaning is ambiguous.
```

Comment:
Er det mer korrekt å si "as its meaning is complicated" i stedet for ambiguous? Er vel _nokså_ entydig, bare veldig forvirrende? 👀